src/werkzeug/debug/shared/debugger.js

```diff
@@ -1,3 +1,25 @@
+function fadeOut(el){
+    el.style.opacity = 1;
+
+    (function fade() {
+        if ((el.style.opacity -= .1) < 0) el.style.display = "none";
+        else requestAnimationFrame(fade);
+    })();
+};
+
+function fadeIn(el, display){
+    el.style.opacity = 0;
+    el.style.display = display || "block";
+
+    (function fade() {
+        let val = parseFloat(el.style.opacity);
+        if (!((val += .1) > 1)) {
+            el.style.opacity = val;
+            requestAnimationFrame(fade);
+        }
+    })();
+};
```

Comment:
No semi-colon for function definitions.

Reply:
Same thing here, trying to clean up the parenthesis usage - can we use `(val + .1 <= 1)` instead of `(!((val += .1) > 1))`

Reply:
Or if `val` needs to be changed:
```
let val =  parseFloat(el.style.opacity) + .1;
if (val <= 1) {
 ...
}
```